packages/react-app/src/graphql/subgraph.ts

```diff
@@ -84,13 +84,39 @@ export interface Player {
   id: string
   amountBet: BigNumberish
   pricesReceived: BigNumberish
+  totalAttributions: BigNumberish
 }
 
+
 export const PLAYER_FIELDS = `
   fragment PlayerFields on Player {
     id
     amountBet
     pricesReceived
+    totalAttributions
+  }
+`;
+
+export interface Attribution {
+  id: string
+  market: {
+    id: string,
+    name: string
+  }
+  amount: BigNumberish
+  attributor: {
+    id: string
+  }
+  timestamp: BigNumberish
```

Comment:
timestamp is missing in ATTRIBUTION_FIELDS

Reply:
timestamp right now it's only used for ordering, but adding the field to be consistent